packages/vite-plugin-docs/docs/getting-started/vue/00-create-project.md

```diff
@@ -42,7 +42,7 @@ import { defineConfig } from 'vite'
 import vue from '@vitejs/plugin-vue'
 // highlight-start
 import { crx } from '@crxjs/vite-plugin'
-import manifest from './manifest.json'
+import manifest from './manifest.json' assert { type: 'json' }
```

Comment:
CRXJS supports Node >=14, so unfortunately the docs need to support both ways of importing JSON. What do you think about this approach?

```suggestion
import manifest from './manifest.json' // Node 14 >= 16
import manifest from './manifest.json' assert { type: 'json' } // Node >= 17
```

Reply:
@jacksteamdev you're the boss, however I first tried with Node 14 and JSON import didn't work, then I looked up how to import and landed on resource hits. Maybe I didn't import correctly during Node 14... 
if that's indeed the case, then yeah this makes sesne